### ISSUE TITLE
[Bug]  Image option in the widget settings schema doesnot work.

### ISSUE THREAD

**Issue by lsyer** [thingsboard/thingsboard#9929]
**Describe the bug**
In version 3.6.3, when using the image option in the widget settings schema, the widget's "Add" button becomes unresponsive when used in the dashboard. 

Even for older widgets that have been directly upgraded from version 3.4.2, they can still function, but they are limited to fetching images with the address starting with "tb-image:" which is used for embeded in "image.embed-to-angular-template". 

This bug results in the malfunctioning of widgets that previously used direct URLs.

**Your Server Environment**

* own setup
  * Ubuntu 20.04, Deb
  * ThingsBoard Version: 3.6.3

**To Reproduce**
1. Import the attached JSON in Widgets library
2. Open any dashboard
3. Step into edit mode and add the imported widget
4. Change the bg image, oh, we can see it

**Screenshots**
![2024-01-03_01-37](https://github.com/thingsboard/thingsboard/assets/33717782/d2d832b0-b10f-4a7b-8f3d-0517ffe18af3)
![2024-01-03_02-02](https://github.com/thingsboard/thingsboard/assets/33717782/3f2ea2b0-1761-4831-859d-c6dcad2c7b4c)

**Additional context**
Widget exported JSON:
[bgimg.json](https://github.com/thingsboard/thingsboard/files/13812699/bgimg.json)

**Comment by vvlladd28** [comment]
Hi @lsyer,
Thanks for your report, we will fix it in the next release

**Comment by lsyer** [comment]
And it might be fixed in ui-ngx/src/app/shared/components/json-form/react/json-form-image.tsx, line 77-85, but I didn't find how to convert the tb-image address to real image url, like embeded code shown in image gallery.
![2024-01-06_10-36](https://github.com/thingsboard/thingsboard/assets/33717782/f94d9fcc-24a6-47d2-a4a4-c9cd81ae7cc0)

```
        let previewComponent;
        if (this.state.imageUrl) {
            if (this.state.imageUrl.startsWith(TB_IMAGE_PREFIX)){
                let imageUrl; // = removeTbImagePrefix(this.state.imageUrl)
                // TODO: get the real image url
                previewComponent = <img className='tb-image-preview' src={imageUrl} />;
            } else {
                previewComponent = <img className='tb-image-preview' src={this.state.imageUrl} />;
            }
        } else {
            previewComponent = <div>No image selected</div>;
        }
```

**Comment by lsyer** [comment]
Fixed by #9986
